### PR TITLE
CONTRIBUTING.md: first version

### PR DESCRIPTION
**Pull request by ret2libc**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
-Syntax coding style guidelines
-==============================
+# How to report issues
+
+Before reporting an issue with GitHub, be sure that:
+* you are using the git version of radare2
+* you are using a clean installation
```
Comment (ret2libc) [inline CONTRIBUTING.md]: @radare what are the commands to be sure that you have everything clean? I'd like to add those in this line.

Comment (radare) [inline CONTRIBUTING.md]: make purge PREFIX=/usr

Comment (radare) [inline CONTRIBUTING.md]: `make mrproper` or `git clean -xdf` will clean the srcdir

Comment (ret2libc) [inline CONTRIBUTING.md]: I also use that, but I think that's not completely clean, because shlr/capstone will be kept.

**Comment by radare** [comment]
merged, we can improve it later. thanks!

**Pull request closed without merging**